### ISSUE TITLE
x3d node detach and prepend

### ISSUE THREAD

**Issue by doczoidberg** [x3dom/x3dom#465]
I am detaching and preprending my x3d DOM node via jquery. After that changes on that DOM element are not recognized any more by x3dom. For example changing the url of an inline node doesn't reload the file: 

Html:
```
<x3d id="the_element">   
         <inline id="myElement" url="myurl" ></inline> ....
```

Javscript:
```
$('#the_element').detach().prependTo('#bauteilcontent');
myurl = "/newfile.x3d";
```
